backend/src/main/java/com/soen341/instagram/dto/picture/PictureDTO.java

```diff
@@ -0,0 +1,42 @@
+package com.soen341.instagram.dto.picture;
```

Comment:
Overall I think this is fine. If anything, I would say that the methods could have slightly more specific names to better identify that we are manipulating a picture (getPictureCaption, etc.) for clarity.